blocks/init/src/Blocks/components/icon/components/icon-options.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'; // eslint-disable-line no-unused-vars
+import { __ } from '@wordpress/i18n';
+import { Fragment } from '@wordpress/element';
+import { SelectControl, ToggleControl, Icon } from '@wordpress/components';
+import { ColorPaletteCustom } from '@eightshift/frontend-libs/scripts/components';
+import { checkAttr } from '@eightshift/frontend-libs/scripts/helpers';
+import { getOptionColors, icons } from '@eightshift/frontend-libs/scripts/editor';
+import manifest from './../manifest.json';
+
+const { options, title } = manifest;
+
+export const IconOptions = (attributes) => {
+	const {
+		setAttributes,
+		componentName = manifest.componentName,
+		label = title,
+		iconUse = checkAttr('iconUse', attributes, manifest, componentName),
+		iconName = checkAttr('iconName', attributes, manifest, componentName),
+		iconColor = checkAttr('iconColor', attributes, manifest, componentName),
+		showIconOptions = true,
+	} = attributes;
+
+	if (!showIconOptions) {
+		return null;
+	}
+
+	return (
+		<Fragment>
+			{label && <h3 className={'options-label'}>{label}</h3>}
+
+			<ToggleControl
+				label={__('Use Icon', 'eightshift-frontend-libs')}
```

Comment:
This shouldn't be hardcoded.

Write it like this:
`label={sprintf(__('Use %s', 'eightshift-frontend-libs'), label)}`

Reply:
Good catch